test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -86,11 +86,12 @@ connection_simulator::query_timestamp(
   const std::string &config, std::string &hex_ts, bool &ts_supported)
 {
     std::string query_timestamp;
+    timestamp_manager *ts_manager = &timestamp_manager::get_timestamp_manager();
```

Comment:
Declare it where you use it!

Reply:
I have used it in the `else` part and at the end of the func, so I think we should define it at the top, no?

Reply:
My bad